src/main/java/org/jenkinsci/plugins/vsphere/builders/ReconfigureCpu.java

```diff
@@ -75,6 +81,8 @@ public boolean reconfigureCPU (final Run<?, ?> run, final Launcher launcher, fin
         PrintStream jLogger = listener.getLogger();
         String expandedCPUCores = cpuCores;
         String expandedCoresPerSocket = coresPerSocket;
+        ResourceAllocationInfo resAllInfo = cpuReservation;
```

Comment:
Replace with something like
```
ResourceAllocationInfo resAllInfo = new ResourceAllocationInfo();
cpuReservation.setReservation((long)Integer.valueOf(getCpuLimitMHz()));
```
...except that you'll also need to make sure that, if `cpuLimitMHz` is null or empty (which it can be for folks who aren't using this new capability), you don't call `spec.setCpuAllocation(resAllInfo)` at all.